From d6b1c20af88955ec03713bd8cf7041740286db45 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?M=C3=A5rten=20Kongstad?= <amhk@google.com>
Date: Thu, 19 Feb 2026 20:04:59 +0000
Subject: [PATCH] Fix compilation error after 37.0 finalization

Fix the following error:

  ERROR: [...]/packages/apps/Settings/src/com/android/settings/bluetooth/DeviceListPreferenceFragment.kt:345:16 \
         Only safe (?.) or non-null asserted (!!.) calls are allowed on a nullable receiver of type 'BluetoothLeScanner?'.

Bug: 480974361
Test: N/A
Flag: NONE platform SDK finalization
Change-Id: I57e970528f58ebee7d0155836ed9662edbdbb22b
---
 .../android/settings/bluetooth/DeviceListPreferenceFragment.kt  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/com/android/settings/bluetooth/DeviceListPreferenceFragment.kt b/src/com/android/settings/bluetooth/DeviceListPreferenceFragment.kt
index 2153348df82..f6661729d4d 100644
--- a/src/com/android/settings/bluetooth/DeviceListPreferenceFragment.kt
+++ b/src/com/android/settings/bluetooth/DeviceListPreferenceFragment.kt
@@ -342,7 +342,7 @@ abstract class DeviceListPreferenceFragment(restrictedKey: String?) :
         val settings = ScanSettings.Builder()
             .setScanMode(ScanSettings.SCAN_MODE_LOW_LATENCY)
             .build()
-        scanner.startScan(leScanFilters, settings, leScanCallback)
+        scanner!!.startScan(leScanFilters, settings, leScanCallback)
     }
 
     private fun stopLeScanning() {
-- 
2.53.0.345.g96ddfc5eaa-goog

